### PR TITLE
Task #34166 User edit profile

### PR DESCRIPTION
**Pull request by phuongbk**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -4,9 +4,10 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   protected
-
   def configure_permitted_parameters
     devise_parameter_sanitizer.for(:sign_up) {|u| u.permit(:name, :email,
      :password, :password_confirmation)}
+    devise_parameter_sanitizer.for(:account_update) {|u| u.permit(:name, :email,
```
Comment (leminhtuan2015) [inline app/controllers/application_controller.rb]: Please use do-end if you have 2 line, ({} for one line)

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".title" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name),
+      html: { method: :put }) do |f| %>
```
Comment (leminhtuan2015) [inline app/views/devise/registrations/edit.html.erb]: {} space

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (leminhtuan2015) [inline app/controllers/application_controller.rb]: rm ()

Comment (phuongbk) [inline app/controllers/application_controller.rb]: have error if rm ()

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (phuongbk) [inline app/views/devise/registrations/edit.html.erb]: tks a!

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".title" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name),
+      html: {method: :put}) do |f| %>
```
Comment (yuyuvn) [inline app/views/devise/registrations/edit.html.erb]: remove ()

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (leminhtuan2015) [inline app/controllers/application_controller.rb]: Really ???
```ruby
devise_parameter_sanitizer.for(:account_update) {|u| u.permit :name, :email,
  :password, :password_confirmation, :current_password}
```

Comment (phuongbk) [inline app/controllers/application_controller.rb]: Sr, i think rm (:account_update) -> :account_update

**Pull request merged**